ClinGen/clincoded
# Updated Parsing of ClinVar E-utilities XML (to support VCV format)

Steps to test #2034:
1. Start a new variant curation (based on ClinVar ID).
2. Upon retrieving the variant from ClinVar, check that the **HGVS terms** list includes items identified as GRCh37 and/or GRCh38 (in parenthesis after the term). If such items aren't included, it likely means the data isn't present in ClinVar's E-utilities XML (and a new variant should be selected).
3. Upon saving the variant and being taken to the evidence view, confirm that the "variant is not associated with a GRCh37 genomic representation" warning does not appear.
4. Start the interpretation and confirm there are no problems with existing functionality (adding a disease, inheritance, criteria evaluations, PMID evidence, etc.). If curating as an affiliation, use the approval process to approve the interpretation and then generate ClinVar submission data.
